contracts/core/Allo.sol

```diff
@@ -79,10 +79,6 @@ contract Allo is
     /// @dev 'Pool.id' -> 'Pool'
     mapping(uint256 => Pool) private pools;
 
-    /// @notice Returns a bool for whether a strategy is cloneable or not using the strategy address as the key
-    /// @dev Strategy.address -> bool
-    mapping(address => bool) private cloneableStrategies;
```

Comment:
Removing a mapping disrupts this consistency of the storage layout and could lead to unexpected behavior if we need to add new storage variables in a future upgrade.
Consider keeping it with a comment marking it as deprecated/not used.